src/lrud.test.js

```diff
@@ -587,4 +587,20 @@ describe('lrud', () => {
       expect(navigation.getNode('a').activeChild).toEqual('c')
     })
   })
+
+  describe('regression test', () => {
+    test('going down a row when matching index aligned node is unfocusable', () => {
```

Comment:
Can this test be moved to the `handleKeyEvent() - simple index alignment behaviour` suite in [`index-align.test.js`](https://github.com/bbc/lrud/blob/f739467f55125c89c0ebe4faf4546972f50798b5/src/index-align.test.js#L5)?

Reply:
Done!